measures/ResStockArgumentsPostHPXML/measure.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+# see the URL below for information on how to write OpenStudio measures
+# http://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/
+
+# start the measure
+class ResStockArgumentsPostHPXML < OpenStudio::Measure::ModelMeasure
+  # human readable name
+  def name
+    # Measure name should be the title case of the class name.
+    return 'ResStock Arguments Post-HPXML'
+  end
+
+  # human readable description
+  def description
+    return 'Measure that post-processes the output of the BuildResidentialHPXML and BuildResidentialScheduleFile measures.'
+  end
+
+  # human readable description of modeling approach
+  def modeler_description
+    return 'Passes in all ResStockArgumentsPostHPXML arguments from the options lookup, processes them, and then modifies output of other measures.'
+  end
+
+  # define the arguments that the user will input
+  def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
+    args = OpenStudio::Measure::OSArgumentVector.new
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('hpxml_path', false)
+    arg.setDisplayName('HPXML File Path')
+    arg.setDescription('Absolute/relative path of the HPXML file.')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument('output_csv_path', false)
+    arg.setDisplayName('Schedules: Output CSV Path')
+    arg.setDescription('Absolute/relative path of the csv file containing user-specified occupancy schedules. Relative paths are relative to the HPXML output path.')
+    args << arg
+
+    return args
+  end
+
+  # define what happens when the measure is run
+  def run(model, runner, user_arguments)
+    super(model, runner, user_arguments)
+
+    # use the built-in error checking
+    if !runner.validateUserArguments(arguments(model), user_arguments)
+      return false
+    end
+
+    # assign the user inputs to variables
+    args = get_argument_values(runner, arguments(model), user_arguments)
+    args = Hash[args.collect { |k, v| [k.to_sym, v] }]
+
+    hpxml_path = args[:hpxml_path].get
+    unless (Pathname.new hpxml_path).absolute?
+      hpxml_path = File.expand_path(File.join(File.dirname(__FILE__), hpxml_path))
+    end
+    unless File.exist?(hpxml_path) && hpxml_path.downcase.end_with?('.xml')
+      fail "'#{hpxml_path}' does not exist or is not an .xml file."
+    end
+
+    _hpxml = HPXML.new(hpxml_path: hpxml_path)
+
+    # init
+    new_schedules = {}
+
+    # TODO: populate new_schedules
```

Comment:
Do we need to add the schedules here or is this section just a placeholder for future PRs?

Reply:
It's just a placeholder for future PRs.